src/harness/testcases/heartbeat_testcase.py

```diff
@@ -139,6 +139,94 @@ def test_10_9_4_2_3_1_1(self):
     self.assertEqual(response['response']['responseCode'], 102)
 
   @winnforum_testcase
+  def test_WINNF_FT_S_HBT_10(self):
+    """Initial Heartbeat Request (immediately after CBSD moves
+    into Granted State) is from three CBSDs with an unsupported
+    protocol version by SAS.
+
+    The response should be FAIL, code 100.
+    """
+
+    # Register the devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    self._sas_admin.InjectFccId({'fccId': device_a['fccId']})
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_e.json')))
+    self._sas_admin.InjectFccId({'fccId': device_b['fccId']})
+    device_c = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_c.json')))
+    self._sas_admin.InjectFccId({'fccId': device_c['fccId']})
+    request = {'registrationRequest': [device_a, device_b, device_c]}
+    response = self._sas.Registration(request)['registrationResponse']
+    # Check registration response
+    for resp in response:
+        self.assertEqual(resp['response']['responseCode'], 0)
+    cbsd_ids = (response[0]['cbsdId'], response[1]['cbsdId'], response[2]['cbsdId'])
```

Comment:
Consider either putting this in the 'for' loop on line 163 or the following for compactness/readability:

```
cbsd_ids = [resp['cbsdId'] for resp in response]
```